### ISSUE TITLE
Sprays in menu download again

### ISSUE THREAD

**Issue by LegacyValken** [swampservers/contrib#474]
https://files.catbox.moe/k09jon.mp4

Title. It's been happening for months now, when opening the sprays menu it'll download them and keep them cached for a bit only for them to lose the images and eventually force the player to re-download them again.

I'm not sure if this is on a short timer, but it's become such a nuisance where the images just end up not loading because of their file size or having to wait for each and every spray to load again just so I can select a different one.

I just want my sprays to last a long while on the client cache so I don't have to have them re-download a minute afterwards.

**Comment by WinterPhoenix** [comment]
Okay so what's happening here is there are no "references" to the textures for some amount of time and they get "unloaded." I say unloaded in quotes because while they are unloaded in Lua and the references are removed there, the game still has the textures loaded in memory, so we're effectively just losing track of them.

tl;dr I fixed it
